distribution/docs/src/main/resources/content/_whitelists/spatial-whitelist.adoc

```diff
@@ -6,25 +6,51 @@
 :order: 06
 :summary: ${ddf-spatial} whitelist.
 
-The following packages have been exported by the ${ddf-spatial} application and are approved for use by third parties:
-
-* `net.opengis.cat.csw.v_2_0_2.dc.elements`
-* `net.opengis.cat.csw.v_2_0_2.dc.terms`
-* `net.opengis.cat.csw.v_2_0_2`
-* `net.opengis.filter.v_1_1_0`
-* `net.opengis.gml.v_3_1_1`
-* `net.opengis.ows.v_1_0_0`
-* `org.codice.ddf.spatial.geocoder.geonames`
-* `org.codice.ddf.spatial.geocoder`
-* `org.codice.ddf.spatial.geocoding.context`
-* `org.codice.ddf.spatial.geocoding`
-* `org.codice.ddf.spatial.kml.endpoint`
-* `org.codice.ddf.spatial.kml.transformer`
-* `org.codice.ddf.spatial.ogc.catalog.resource.impl`
-* `org.codice.ddf.spatial.ogc.catalog`
-* `org.codice.ddf.spatial.ogc.wfs.catalog.converter`
-* `org.codice.ddf.spatial.ogc.wfs.catalog.mapper`
-* `org.codice.ddf.spatial.ogc.wfs.v1_0_0.catalog.converter`
-* `org.codice.ddf.spatial.ogc.wfs.v2_0_0.catalog.converter`
+The following classes have been exported by the ${ddf-spatial} application and are approved for use by third parties:
 
+In package `org.codice.ddf.spatial.geocoder`
+
+* `GeoCoder`
+
+In package `org.codice.ddf.spatial.geocoding.context`
+
+* `NearbyLocation`
+
+In package `org.codice.ddf.spatial.geocoding`
```

Comment:
All interfaces in this package are new

Reply:
I don't know for sure, but I think these should have been marked as experimental.

Reply:
❗️If these are not experimental, we need to include `GeoEntry` because it is returned and used by some of the interfaces.

Reply:
Talked with @jrnorth and these were not meant to be in the whitelist. Removing all geocoding packages.